File: ruby/delivery-api/GetTaxonomyGroups.rb
# DocSection: delivery_api_get_taxonomy_groups
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>'
delivery_client.taxonomies
               .limit(3)
               .execute do |response|
                 groups = response.taxonomies
               end
# EndDocSection

File: ts/delivery-api/GetTaxonomyGroups.ts
// DocSection: delivery_api_get_taxonomy_groups
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { createDeliveryClient } from '@kontent-ai/delivery-sdk';

const deliveryClient = createDeliveryClient({
  projectId: '<YOUR_PROJECT_ID>'
});

const response = await deliveryClient.taxonomies()
    .limitParameter(3)
    .toPromise();
// EndDocSection